website/templates/project/edit_draft_registration.mako

```diff
@@ -48,7 +48,7 @@
 
                 <div class="row" style="margin-bottom: 10px;">
                   <span>
-                    Last saved: <span data-bind="text: lastSaveTime"></span>
+                    Last auto-saved: <span data-bind="text: lastSaveTime"></span>
```

Comment:
@brianjgeiger Yes, before making the change I talked to @jinluyuan about this (: